incubator/binding-mqtt/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/internal/stream/MqttServerFactory.java

```diff
@@ -2078,6 +2176,11 @@ private void doSendSessionState(
             long traceId,
             List<String> topicFilters)
         {
+            final MqttDataExFW.Builder sessionDataExBuilder =
+                mqttSessionDataExRW.wrap(sessionExtBuffer, 0, sessionExtBuffer.capacity())
+                    .typeId(mqttTypeId)
+                    .session(sessionBuilder -> sessionBuilder.kind(k -> k.set(MqttSessionDataKind.STATE)));
```

Comment:
Lambda variables should be single letters, such as `s` instead of `sessionBuilder` here.